cowprotocol/cow-amm
# Add GC and Sepolia deployments

Adds new deployment addresses on GC and Sepolia. The deployment was done in a [merge commit](https://github.com/cowprotocol/cow-amm/commit/7df7e5665855fbd90a155827b6dbf44be1189b8b) that includes #10 and #11.

I kept the latest run of the scripts in `broadcast/` and deleted everything else.
The file `networks.json` was built by hand.

### How to test

Check the addresses/transactions in `network.json` and see the verified code on Etherscan.
